docs-sphinx/reference/smart-contract/genesis-contract.md

```diff
@@ -0,0 +1,217 @@
+# Genesis Contract
+
+## Overview
+
+Genesis Contract, also known as the Zero Contract, is mainly used to deploy and maintain smart contracts running on the aelf blockchain.
+
+This contract will be deployed first when aelf blockchain launched so that it can be used to deploy other smart contracts.
+
+To achieve this purpose, the Genesis Contract implements the following methods defined in [acs0](https://docs.aelf.io/en/latest/reference/acs/acs0.html):
+
+```protobuf
+service ACS0 {
+    // Deploy a system smart contract on chain and return the address of the system contract deployed.
+    rpc DeploySystemSmartContract (SystemContractDeploymentInput) returns (aelf.Address) {
+    }
+    
+    // Deploy a smart contract on chain and return the address of the contract deployed.
+    rpc DeploySmartContract (ContractDeploymentInput) returns (aelf.Address) {
+    }
+    
+    // Update a smart contract on chain.
+    rpc UpdateSmartContract (ContractUpdateInput) returns (aelf.Address) {
+    }
+    
+    // and others.
+    // ...
+}
+```
+
+Therefore, developers can deploy (and update) their own smart contracts by interacting with the Genesis Contract.
+
+In this article, we will discuss:
+
```

Comment:
Add a point here for ## `SmartContractRegistration` and `ContractInfo`, suggestion:
- Understanding message structure and how contract information is stored on chain

Can find something simpler and shorter.